index.js

```diff
@@ -5,13 +5,15 @@ const messages = document.getElementsByClassName('message');
 const tooHighMessage = document.getElementById('too-high');
 const tooLowMessage = document.getElementById('too-low');
 const maxGuessesMessage = document.getElementById('max-guesses');
-const numberOfGuessesMessage = document.getElementById('num-of-guesses');
+const numberOfGuessesMessage = document.getElementById('number-of-guesses');
 const correctMessage = document.getElementById('correct');
+const belowZeroMessage = document.getElementById('below-zero');
+const aboveHundredMessage = document.getElementById('above-hundred');
 
 let targetNumber;
-const attempts = 0;
-const maxNumberOfAttempts = 5;
 
+let maxNumberOfAttempts = 5;
```

Comment:
Nitpick: This number should be a constant because it never decreases. For example, the max attempts for each "game" will always be 5 regardless of how many attempts were made. 

Perhaps this variable has something to do with the "reset" bug below!? Maybe it does! Maybe doesn't! 🤠